hack/verify-crdgen.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+echo "SCRIPT_ROOT: ${SCRIPT_ROOT}"
+source "${SCRIPT_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+
+#CRD_OPTIONS="crd:trivialVersions=true,preserveUnknownFields=false"
+CRD_OPTIONS="crd:trivialVersions=true,preserveUnknownFields=false"
+
+# Create a temp directory
+_tmpdir="$(mktemp -d "/tmp/sched-plugins.XXXX")"
+# trap "rm -rf ${_tmpdir}" EXIT
+
+# Download controller-gen locally
+CONTROLLER_GEN="${GOPATH}/bin/controller-gen"
+$(go install sigs.k8s.io/controller-tools/cmd/controller-gen@latest)
```

Comment:
Why embed it here? We should let the error to be yielded if there is any.

Reply:
Here, did you mean remove the `$()` enclosing for the `go install` command and let it run as it is?